src/components/gltf-model.js

```diff
@@ -22,7 +22,7 @@ module.exports.Component = registerComponent('gltf-model', {
     this.remove();
 
     this.loader.load(src, function gltfLoaded (gltfModel) {
-      self.model = gltfModel.scene;
+      self.model = gltfModel.scene || gltfModel.scenes[0];
```

Comment:
If `gltfModel.scenes[0]` exists, then `gltfModel.scene || gltfModel.scenes[0]` will always be `gltfModel.scene`.